### DISCUSSION TITLE
Troubleshooting with Blender Crashing

### DISCUSSION THREAD

**Discussion by TeddyBearJosh** [skyslide22/blender-addon-for-trackmania-and-maniaplanet#26] [Q&A]
Hello, I need a little bit of help here. I'm trying to export a cube (yes a cube with 6 faces) as a test, and I am following all the documentation and tutorial videos as best I can, yet every time I try to export the object inside of the collection it crashes Blender.

(3.31LTS and Blendermania version 3.2.0)

Almost as soon as I click "Export & Convert", it crashes Blender. I'll spend tomorrow trying different methods and different software versions so hopefully that fixes it. Any help would be greatly appreciated as a map project I'm working on will need better modelling tools than what Trackmania Editor currently has, and I have experience with Blender.

**Comment by TeddyBearJosh** [comment]
Going through the files, it seems to be creating the FBX and XML files correctly (Trackmania\Work\Items) but failing to create the GBX file (Trackmania\Items) thus resulting in a crash.

**Reply by skyslide22** [reply]
could you open blender from the terminal and tell me what the last messages are?
should look something like this:
<kbd>win+r</kbd> then write `cmd` then put `%USERPROFILE%\AppData\Roaming\Blender Foundation\Blender\3.3\blender.exe` or similar

**Reply by TeddyBearJosh** [reply]
![image](https://user-images.githubusercontent.com/67753340/194710463-61adf87d-89f6-40b8-82d2-db2a36b83459.png)
Thanks for the quick reply! I think I need to change some privacy settings to allow it to run scripts (not sure why its disabled)
Looking through my privacy settings, everything seems to be default and open.

**Reply by TeddyBearJosh** [reply]
[testexport.crash.txt](https://github.com/skyslide22/blendermania-addon/files/9739396/testexport.crash.txt)
